diff --git a/node_modules/mathjax-react/dist/index.es.js b/node_modules/mathjax-react/dist/index.es.js
index 283003b..8e77cf9 100644
--- a/node_modules/mathjax-react/dist/index.es.js
+++ b/node_modules/mathjax-react/dist/index.es.js
@@ -222,7 +222,7 @@ var MathComponent = /** @class */ (function (_super) {
           console.log('Didn\'t work!', this.rootRef);
         }
         */
-        return this.props.display ? (createElement("div", { ref: this.rootDivRef, dangerouslySetInnerHTML: { __html: renderedSVG } })) : (createElement("span", { ref: this.rootSpanRef, dangerouslySetInnerHTML: { __html: renderedSVG } }));
+        return this.props.display ? (createElement("div", { ref: this.rootDivRef, dangerouslySetInnerHTML: { __html: renderedSVG } })) : (createElement("title", { ref: this.rootSpanRef, dangerouslySetInnerHTML: { __html: renderedSVG } }));
     };
     MathComponent.defaultProps = defaultProps; // bind default props
     return MathComponent;
